torchgeo/datamodules/geo.py

```diff
@@ -30,7 +30,7 @@ class GeoDataModule(LightningDataModule):  # type: ignore[misc]
     """
 
     mean = torch.tensor(0)
-    std = torch.tensor(255)
+    std = torch.tensor(1)
```

Comment:
I actually kind of like this change, although we might want to override the default in other datamodules. Might want to check the PR where I introduced this class to see which datamodules previously used 255.

Reply:
Seems like `DeepGlobeLandCover`, `GID15`, `Vaihingen` and `Potsdam2D` are the ones that used 255. Might move this to a separate PR.

Reply:
Let's separate this out